Portfolio: Desktop version finish.

Zafron047 opened the pull request:
- [ ] Added desktop based html 
- [ ] Added desktop based css design

Zafron047 committed:
add html css and missing Images

huseyinbiyik commented on style.css:
```diff
@@ -7,458 +7,1006 @@
-  border: none;
+/* media query for mobile version  */
+
+@media only screen and (max-width: 768px) {
```
Since we follow the mobile first, kindly remove this media query to make your code following DRY practice. In this way, you will not have a repeating code. For example, you have this code under max-width: 768px.
```css
  .head h1 {
    font-style: normal;
    font-weight: 700;
    font-size: 40px;
    line-height: 52px;
    margin-bottom: 12px;
    display: flex;
    align-items: center;
    color: #172b4d;
  }
``` 
But in min-width: 768 you have the same code again. Only one line is different. 
If you wouldn't use max-width:768px; , you would only have on the desktop version (min-width:768px.)
```css
  .head h1 {
font-size: 48px;
}
```

Zafron047 replied:
Done

huseyinbiyik commented on style.css:
```diff
@@ -7,458 +7,1006 @@
-  margin-top: 36px;
-  border-radius: 10px;
-  border: none;
+/* media query for mobile version  */
```
Your mobile version looks broken right now. Kindly fix the horizontal overflow issue, fix your cards and social links on the About Myself section. Basically, it needs to be the same as your 'main' branch. It was just perfect 👍 
![l](https://user-images.githubusercontent.com/25887253/230667902-c1f57873-2302-4865-a5ca-7670e9d195b6.gif)
<img width="422" alt="Screenshot 2023-04-07 at 10 38 48 PM" src="https://user-images.githubusercontent.com/25887253/230667953-fba07caf-f687-4556-b104-6295227d1663.png">
<img width="422" alt="Screenshot 2023-04-07 at 10 39 27 PM" src="https://user-images.githubusercontent.com/25887253/230667960-00e433c4-f870-478e-8487-d9750df5f0b1.png">

Zafron047 replied:
Hi @huseyinbiyik thanks for the hardwork reviewing my codes.

Yes, while making the mediaquery for desktop version I have somehow messed the classes of mobile version.
I still just to submitted the desktop version while I can find out what made classes in mob first got messed up.

I have one curious question though. Is mobile version included in the core requirements for this project??

As I have double checked, this project will only be based on desktop version?!! Otherwise I would refrain from submitting.
If you say I will not submit again before fixing all the versions.

Additional advice on mobile version will also help a lot 🙂 thanks.

Waiting for your advice.

Thanks.

huseyinbiyik replied:
Hello @Zafron047,

We are building our projects on top of each other, so we are developing them instead of breaking the previous ones and I think it is common sense. But if you still think the review is unfair please ask for second opinion with [this form](https://airtable.com/shrQAqnBwek5a0O0s). 👍

Zafron047 replied:
@huseyinbiyik I don't think the code was unfair. 
You have done a thorough inspection which is very useful for me. And I appreciate it.

However, I just followed the necessary requirements documented for submission.

I need to fully understand the requirements again since we have limited reviews keeping it in mind if it will get rejected outside the desktop version again.

And 🤔, as I came to know both versions will be checked in next block's project where we will have to debug this project?

Further advice please.
Should I reuse the previous css stylings for the mobile version, And only change the desktop stylings to fix both issues?

Zafron047 replied:
Fix Mobile version done.

huseyinbiyik commented on style.css:
```diff
@@ -7,458 +7,1006 @@
+    display: none;
+  }
+
+  .cards-row {
```
Great job using flexbox here. 👍  But cards have two sections, one is for description and the other one is for image. And they should use an equal width (half-half) like the Figma design guideline. On yours, description area has less space than image. Kindly fix that 👍 
<img width="805" alt="Screenshot 2023-04-07 at 10 22 32 PM" src="https://user-images.githubusercontent.com/25887253/230668167-54efb0aa-b251-4a08-a17c-097aea474455.png">
<img width="1157" alt="Screenshot 2023-04-07 at 10 22 50 PM" src="https://user-images.githubusercontent.com/25887253/230668171-b9aec5cf-1dfd-4389-b2cc-1c7be4e80a96.png">

Zafron047 replied:
Made cards both parts 50 & 50%

huseyinbiyik commented on index.html:
```diff
@@ -1,268 +1,290 @@
+                            <li>
+                                <div>
+                                    <div class="arrow">
+                                        <p>Languages</p>
```
Please fix the Languages, Frameworks and Skills headings' font-size, font-weight and font-color.
<img width="420" alt="Screenshot 2023-04-07 at 10 26 45 PM" src="https://user-images.githubusercontent.com/25887253/230668791-9fc1d1bd-4424-445b-b263-41b876b201f9.png">
<img width="420" alt="Screenshot 2023-04-07 at 10 28 16 PM" src="https://user-images.githubusercontent.com/25887253/230668793-173465dc-1d12-4e04-9772-62ae39b4e4c0.png">

huseyinbiyik commented on index.html:
```diff
@@ -1,268 +1,290 @@
-                <button>See project</button>
-            </div>
+                        <div class="project-button">
+                            <button>See project</button>
```
Please fix the color of your button's states. Hover and pressed(active) states needs to be fixed according to the Figma guideline.
<img width="463" alt="Screenshot 2023-04-07 at 10 25 18 PM" src="https://user-images.githubusercontent.com/25887253/230669220-99733b85-96f5-4c1d-a76c-b9c63a578b4f.png">
<img width="420" alt="Screenshot 2023-04-07 at 10 25 35 PM" src="https://user-images.githubusercontent.com/25887253/230669224-70b4a77d-9df2-4506-a5ad-adc6ab6c7fec.png">

Zafron047 replied:
Languages, Frameworks and Skills headings' font-size, font-weight and font-color fixed.

huseyinbiyik commented on style.css:
```diff
@@ -7,458 +7,1006 @@
+    padding: 1rem 0;
+  }
+
+  .tech {
```
Please make sure you are following the Figma design guideline and fix font-weight, font-size and font-color of the language.
<img width="420" alt="Screenshot 2023-04-07 at 10 29 06 PM" src="https://user-images.githubusercontent.com/25887253/230669400-076a718c-2796-4258-a599-e14566909645.png">
<img width="535" alt="Screenshot 2023-04-07 at 10 30 12 PM" src="https://user-images.githubusercontent.com/25887253/230669403-b510437b-5ccf-46d6-b4fa-50e6aeb2a585.png">

Zafron047 replied:
fix font-weight, font-size and font-color of the language.
DONE.

huseyinbiyik commented on style.css:
```diff
@@ -7,458 +7,1006 @@
+    gap: 12px;
+  }
+
+  .sct-1 h2 {
```
Please fix the font color and size of your h2 element here.
<img width="649" alt="Screenshot 2023-04-07 at 10 32 30 PM" src="https://user-images.githubusercontent.com/25887253/230670150-be512205-92c2-4e1d-b6dd-e1cd465169e5.png">
<img width="279" alt="Screenshot 2023-04-07 at 10 33 05 PM" src="https://user-images.githubusercontent.com/25887253/230670163-aa8884a2-4bd4-478d-b606-02799fe92fd5.png">

Zafron047 replied:
Fixed the font color and size of your h2 element here.

huseyinbiyik commented on index.html:
```diff
@@ -1,268 +1,290 @@
+                <div class="abt-desk">
+                    <div class="sct-1">
+                        <h2>About Myself</h2>                
+                        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```
Kindly fix the font-size of this text. We need to follow the Figma design guideline for each CSS values 👍 
<img width="279" alt="Screenshot 2023-04-07 at 10 33 18 PM" src="https://user-images.githubusercontent.com/25887253/230671150-b17a5057-16d9-477f-809b-8b39016240c9.png">
<img width="784" alt="Screenshot 2023-04-07 at 10 34 20 PM" src="https://user-images.githubusercontent.com/25887253/230671162-eea50099-1d2e-443d-b01d-f64833b59000.png">

huseyinbiyik reviewed (changes requested):
Hi @Megagig 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![Almost there](https://media.giphy.com/media/Rv6GGaNrRbtNConlYo/giphy.gif)

## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌

## Required Changes ♻️
-  Please make sure your commit messages are meaningful and descriptive for your next commits. Use imperative tense and capitalize the first word. For example, 
👌"Add logo to navigation bar"
👌"Fix close button functionality on mobile navbar"
❌ "Added logo to navigation bar"
❌ "Added HTML"
_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Zafron047 committed:
Fix Mobile version

Zafron047 committed:
Fix Desktop Version

in this I have fixed the issues raised by the code reviewer.

Zafron047 commented:
@huseyinbiyik Dear code reviewer I have made all the necessary changes you wanted me to.
It was a great experience unfortunately I am having technical problem with my Git so I couldn;t push it earlier.

Hopefully I do not have to repeat this module again for technical fault in git hub. 🥲 

Made cards both parts 50 & 50%
Languages, Frameworks and Skills headings' font-size, font-weight and font-color fixed.
fix font-weight, font-size and font-color of the language.
DONE.
Fixed the font color and size of your h2 element here.
Fixed the font-size of this text in About Myself.

AbelG101 commented on style.css:
```diff
@@ -1,464 +1,1083 @@
+    text-decoration: underline;
+  }
+
+  .project-button button:hover {
+    color: #fff;
+    background-color: #6070ff;
+    cursor: pointer;
+  }
```
- [ ] As requested by the previous reviewer please make sure you've also implemented the `:active` or pressed state of the button as shown in the design.

> Please fix the color of your button's states. Hover and pressed(active) states need to be fixed according to the Figma guideline. <img alt="Screenshot 2023-04-07 at 10 25 18 PM" width="463" src="https://user-images.githubusercontent.com/25887253/230669220-99733b85-96f5-4c1d-a76c-b9c63a578b4f.png"> <img alt="Screenshot 2023-04-07 at 10 25 35 PM" width="420" src="https://user-images.githubusercontent.com/25887253/230669224-70b4a77d-9df2-4506-a5ad-adc6ab6c7fec.png">

Zafron047 replied:
Fixed.

AbelG101 commented on style.css:
```diff
@@ -1,464 +1,1083 @@
+  .sct-1 h2 {
+    font-size: 40px;
+    line-height: 52px;
+    background-color: #091e42;
```
- [ ] It seems like the changes you made earlier have created a small issue in the heading here please make sure to fix it as it making your portfolio look unprofessional.
    - Here's how it looks:
      ![image](https://user-images.githubusercontent.com/104968236/230733114-243d365f-4ec0-464f-bf49-f8520d7527eb.png)

Zafron047 replied:
Fixed.

AbelG101 commented on style.css:
```diff
@@ -1,464 +1,1083 @@
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    background: #f7f7f9;
```
- [ ] The same thing is happening here with the previous changes creating some issues so please make sure to remove the background color as it doesn't exist in the design:
       - Here's how it looks on your design:
       ![image](https://user-images.githubusercontent.com/104968236/230733232-46d8bc13-2341-4aae-b6be-99f649b49c18.png)

Zafron047 replied:
Solved.

AbelG101 commented on style.css:
```diff
@@ -1,464 +1,1083 @@
-  border-radius: 10px;
-  border: none;
-}
+* {
```
- [ ] As suggested by the previous reviewer you have to make sure the mobile version is working properly as well although it's not specified explicitly in the project requirements, you have to make sure as a developer the new changes you introduce don't break the previous one and as your code reviewer, we are obliged to check that. So please make sure everything looks as it should in the mobile:
     - Here's how it looks now:
        ![firefox_oUd5As6C9m](https://user-images.githubusercontent.com/104968236/230733700-ae92b5a0-e614-4c6b-b2cd-fc14cfb99c80.png)
     - As you can see the project description text is looking very large
        ![firefox_7Wgp7f4WcK](https://user-images.githubusercontent.com/104968236/230733766-e225b08d-7f56-4655-9e10-67f749445902.png)
     - As you can see the project image is misaligned with the description, the image should've been on top.
     ![firefox_KXRS6OXZdM](https://user-images.githubusercontent.com/104968236/230733838-e9a211ca-d021-4d1e-90a4-747fe5b4213b.png)
     - As you can see the "get my resume" button is not looking as it should be in the design.
- [ ] So please make sure all these things are working ✔.

Zafron047 replied:
Solved.

AbelG101 commented on style.css:
```diff
@@ -1,464 +1,1083 @@
+  text-align: center;
+}
+
+.form-bottom form {
+  display: flex;
+  flex-direction: column;
+}
```
- [ ] [OPTIONAL] The previous reviewer forgot to mention this but the form elements aren't aligned as they're shown in the Figma design so please make sure you fix that:
    - Your design:
      ![image](https://user-images.githubusercontent.com/104968236/230734018-95dabcc3-460c-447b-8ef8-b79b98709f6a.png)
    - Figma design:
      ![image](https://user-images.githubusercontent.com/104968236/230734058-dcbb7a86-2c7d-47a9-ade1-0c1819f8b637.png)

AbelG101 reviewed (changes requested):
![](https://media.giphy.com/media/ReyD4OdRiHeLDCRaEP/giphy.gif)

# Hi @Zafron047 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- I know it can be frustrating sometimes but please make sure you've implemented all the requested changes properly. You just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Zafron047 committed:
Fix recommended changes

Mob version and Desktops version issues.

binhussen commented on index.html:
```diff
@@ -1,268 +1,292 @@
+        <section class="container">
+            <header class="header">
+                <div class="header-left">
+                    <img class="logo" src="Images/my logo.jpg" alt="My logo" />
```
- [Optional] Kindly, you mistype the image URL, Please fix it as it has space between `my` and `logo` you have to write it like `Images/My%20Logo.jpg` to display the image, but next time  I will suggest you to use only small case and remove space for static file and directory to follow best practice and protect your self from this type of confusion :+1:

Zafron047 replied:
@binhussen binhussen
Thanks a lot reviewer. Till now all the reviewers only told me that I have a problem with my images you gave me the solution. I will use your advice from now on. 😃

binhussen reviewed (approved):
Hi @Zafron047 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉
<img src="https://user-images.githubusercontent.com/65852621/133696182-6c5b784b-0083-427a-b24d-3ec870915f79.jpg"/>
## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
